src/components/header/Header.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <v-container>
+    <v-row class="-centtexter">
+      <v-col class="mb-4">
+        <h1 class="title-todo">
+          todos
+        </h1>
+      </v-col>
+    </v-row>
+    <TodoFooter />
```

Comment:
The `Header` component shouldn't contain the `TodoFooter`, please move it from here to the `HomePage`